### DISCUSSION TITLE
Any plan to merge feat(Textbox): min/max width ?

### DISCUSSION THREAD

**Discussion by HesamSoleymani** [fabricjs/fabric.js#9399] [General]
https://github.com/fabricjs/fabric.js/pull/8470

I've seen previews @ShaMan123 has provided in the pr above. They were great, exactly what we need in our project.
I want to ask if there is any plan for merging that in near future maybe? This will be great :)

**Comment by asturur** [comment]
We have an hold off on any refactor and changes because this TS migration we started nearly 2 years ago isn't ready to go out yet as stable.
When is out we need to revisit changes in a way that they are as less breaking as possible.
Limiting someting in the case the user is doing something else is a UX problem that often developers summarize with 'i need a max width' while then the problem is way more complex and once maxWidth is in, the next five layers of issues comes out:
- how do i wrap text with a custom behaviour now
- how do i differentiate wrapping when i hit maxWidth
- how do i hide text
- how do i know i m hitting maxWidth, i need an event
So i m trying to make clear we need to have less features and a more smart way to add custom behaviours.
